doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -104,40 +107,80 @@ execution and result reporting along with required artifacts and environments in
 place.
 
 #### 5.3.1 Test Case Model
+As there are more number of VNF at different levels of networking such as access,
+transport and core level as well as OSI level L0-L7. Every network functions

```

Comment:
"Every network functions provides .." change to "Every network function provides .."

Reply:
Updated.  Resolved.